### DISCUSSION TITLE
Automation - create a new row in a different table

### DISCUSSION THREAD

**Discussion by aidenmitchell** [Budibase/budibase#5097] [Help] [answered]
![image](https://user-images.githubusercontent.com/30846409/160150826-cf15d3de-e7c8-4514-832d-29f7d624e40a.png)
When I try this, the Audit Log table is not changed or updated. Can I define how the row is created?

**Comment by melohagan** [comment, marked as answer]
Hi @aidenmitchell 

Your Audit Log table appears to have no columns. You should have a form that looks something like:

![Screenshot 2022-03-28 at 11 01 34](https://user-images.githubusercontent.com/101575380/160374988-dacc1fa3-e912-4e6e-8365-b602b50ebaae.png)
